subsys/bluetooth/mesh/brg_cfg_srv.c

```diff
@@ -30,6 +31,12 @@ static int brg_cfg_srv_init(const struct bt_mesh_model *model)
 	return 0;
 }
 
+void brg_cfg_srv_reset(const struct bt_mesh_model *model)
```

Comment:
Why is not called table initialization when bridge configuration server is initialized?

Reply:
What if initialization is called, but reset isn't?

Reply:
"reset" calls, init states and then delete the values. If init is called, but not reset, then only 'init' will happen (like in usual boot). I didn't get your point.

Reply:
`brg_cfg_srv_init()` is updated to prevent reinitialization by mistake. So, if bridge cfg is already initialized, it must be resetted first.

Reply:
It is quite weird when server is initialized it relies on third party functionality to initialize the table despite that the exact server requires bridging table. It seems architecturally wrong.

Reply:
Stack can be reset only in two ways, reboot or node_reset. So, if you see, init function is removed now. It is enough to have reset function.